### DISCUSSION TITLE
datasource - https://pagure.io/pungi

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#24291] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'd like the project release to be bumped by renovate

https://pagure.io/pungi/releases

### What you tried so far.

The following configuration didn't work:

```
 "regexManagers": [
    {
      "customType": "regex",
      "fileMatch": [
        "^pungi\\/Containerfile$"
      ],
      "matchStrings": [
        "ENV PUNGI_VERSION=\"(?<currentValue>.*?)\"\\s"
      ],
      "depNameTemplate": "pungi",
      "registryUrlTemplate": "https://pagure.io/",
      "datasourceTemplate": "git-tags"
    }
  ]
```

Another option I can think of is the ability to clone the repo before the version check and then lookup the git tags. Is that even possible?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
`git-tags` have no registry URL - you need to put the full git repository URL as the `packageName`. e.g. something like this:

```
 "regexManagers": [
    {
      "customType": "regex",
      "fileMatch": [
        "^pungi\\/Containerfile$"
      ],
      "matchStrings": [
        "ENV PUNGI_VERSION=\"(?<currentValue>.*?)\"\\s"
      ],
      "depNameTemplate": "pungi",
      "packageNameTemplate": "https://pagure.io/pagure",
      "datasourceTemplate": "git-tags"
    }
  ]
```

You can confirm that a repo/url is valid for this by running `git ls-remote <url>` from a non-git local directory.

**Reply by lmilbaum** [reply]
Thanks @rarkins.
config
```
 "regexManagers": [
    {
      "customType": "regex",
      "fileMatch": [
        "^pungi\\/Containerfile$"
      ],
      "matchStrings": [
        "ENV PUNGI_VERSION=\"(?<currentValue>.*?)\"\\s"
      ],
      "depNameTemplate": "pungi",
      "registryUrlTemplate": "https://pagure.io/pungi",
      "datasourceTemplate": "git-tags"
    }
  ]
```
result:
```
DEBUG: Datasource unknown error (repository=lmilbaum/auto-toolchain-containers)
       "datasource": "git-tags",
       "packageName": "pungi",
       "err": {
         "task": {
           "commands": ["ls-remote", "pungi"],
           "format": "utf-8",
           "parser": "[function]"
         },
         "message": "fatal: 'pungi' does not appear to be a git repository\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists.\n",
```

**Reply by rarkins** [reply]
Compare your config to mine..

**Reply by lmilbaum** [reply]
Yup. Found my problem. :-)
The only change I did was from
"packageNameTemplate": "https://pagure.io/pagure"
to
"packageNameTemplate": "https://pagure.io/pungi"

**Reply by rarkins** [reply]
Your last pasted config had two problems (registry url and package name)

**Reply by lmilbaum** [reply]
Correct. My problem is that I can't figure out yet when to use which config key. It doesn't sync in.

**Comment by lmilbaum** [comment]
Now there is a second problem. The new version is not identified.

**Reply by lmilbaum** [reply]
Fixed that as well :-)
